ppr-ui/src/main.ts

```diff
@@ -58,6 +59,15 @@ async function start () {
     await initLdClient()
   }
 
+  // For now, ONLY run for local development
+  // In your local .env add: VUE_APP_LOCAL_DEV="true"
+  if (process.env.VUE_APP_LOCAL_DEV === 'true') {
```

Comment:
As in the comment, if you want to run this locally, just add the env variable to your local env.